src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -7,7 +7,13 @@ namespace System.Runtime.InteropServices
 {
     public static partial class RuntimeInformation
     {
+#if netcore50aot
+        private const string FrameworkName = ".NET Native";
+#elif net46
```

Comment:
This should work on down level .net (4.5.1?) though I guess it's out of support soon